eng/Versions.props

```diff
@@ -4,12 +4,21 @@
   <PropertyGroup>
     <UsingToolNetFrameworkReferenceAssemblies>true</UsingToolNetFrameworkReferenceAssemblies>
   </PropertyGroup>
-  <!-- This is a fake version to fool arcade. In this repo, we handle the versioning
-  of assets ourselves, since all we do is generate zips/tarballs and native installers.
-  To actually update the version for this repo, look at src/redist/targets/Versions.targets -->
   <PropertyGroup>
-    <VersionPrefix>1.0.0</VersionPrefix>
-    <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
+    <VersionMajor>5</VersionMajor>
+    <VersionMinor>0</VersionMinor>
+    <VersionSDKMinor>1</VersionSDKMinor>
```

Comment:
Separate, but we might consider calling this VersionFeature as that is what the global.json roll-forward officially called this.

Reply:
I will change this.